global/scripts/mixtureModel.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+# mixtureModel <inFile> <outFile> [wbcmd] [melocmd] [wbshrtctscmd]
+# Performs Gaussian mixture modeling on precomputed ICs
+# Wrapper around melodic. Accepts nifti or cifti inputs.
+# Require Input:
+#   inFile  : file path to IC z-scores, file path including extension as string (accepts nifti or cifti files)
+#   outFile : file path to IC z-scores with Gaussian mixture modeling
+#              Can output nifti, nifti-gz, or cifti, depending on extension: .nii, .nii.gz, .dscalar.nii,
+#              Output format does not have to match input format, unless output is cifti.
+# Optional Inputs
+#   wbcmd   : workbench command, defaults to 'wb_command'
+#   melocmd : melodic command, defaults to 'melodic'
+#   wbshrtctscmd: workbench shortcuts command, defaults to 'wb_shortcuts'
+#
+# See also:
+# https://www.jiscmail.ac.uk/cgi-bin/webadmin?A2=fsl;6e85d498.1607
+# https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/MELODIC#Using_melodic_for_just_doing_mixture-modelling
+# https://www.fmrib.ox.ac.uk/datasets/techrep/tr02cb1/tr02cb1.pdf
+#
+# Created by Burke Rosen
+# 2024-07-08
+# Dependencies: workbench, FSL
+# Written with workbench 2.0 and FSL 6.0.7.1
+# Refactored from matlab to bash 2025-05-19 by Copilot GPT-4.1
+#
+# ToDo:
+# Feed arbitrary melodic arguments.
+
+mixtureModel() {
+  local inFile="$1"
+  local outFile="$2"
+  local wbcmd="${3:-wb_command}"
+  local melocmd="${4:-melodic}"
+  local wbshrtctscmd="${5:-wb_shortcuts}"
```

Comment:
Scripts in `global/scripts` should probably be using `newopts.shlib`, not positional parameters.